v4/source/elasticsearch.yaml

```diff
@@ -0,0 +1,19 @@
+  /elasticsearch/test:
+    post:
+      tags:
+        - elasticsearch
+      summary: Test Elasticsearch configuration
+      description: |
+        Test the current Elasticsearch configuration to see if the Elasticsearch server can be contacted successfully.
```

Comment:
Actually maybe `**Minimum server version: unreleased**` or something would be better to start with